### PR TITLE
Removal of Gun & Ammo Bloat

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
This PR removes tons of unused guns that were for custom loadouts, various non-canon / non-lore friendly firearms, and other redundant firearms that crowd loot pools. The more guns we have the harder it commonly is to learn what gun does what, what gun is more effectively, less effective, etc.

Also as a test, re-added the AK-112 and Assault Carbine sprites from DR / Atom Bomb to see how they look alongside other firearms. Can be used in the future to port 'larger' sized weapons; if not, we can try the Mojave Sun tactic where we make guns scale to different sizes in world vs icon if needed.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog
:cl:
removes: .38 caliber and ammo.
removes: Various rifles, pistols, and other quirky guns that are not lore-friendly or otherwise redundant/useless.
adds: Few new lore-friendly/canon weapons, mostly akin to ones we once had on DR or other F13 servers.
/:cl:

### CODE REVIEW

**Comment by Stutternov** [comment]
Right - PR is now tested. Not extensively, but I can confirm that the guns added and sprites are loading correctly; loot pools I did try to test a bit but are hard to judge. 

They'll be easy to change though with player feedback if 'out of tier' weapons are spawning or if a default path shows up due to bad pathing.

**Comment by BadAtThisGame302** [comment]
Right well just two questions. 
Did you remove the level-action or is that a unique one? 
Why did you remove the AER12 and not the AER14? The AER14 is a unique NV weapon and the AER12 would probably be more common, even if it is only mentioned in Fallout 3. IMO.

**Comment by Zero-Percent-Angel** [comment]
Okay uh, do we not want any lever action rifles at all? Honestly when it comes to what guns are in and out I am not precious so I'd rather get a community temperature feeling on this

**Comment by Stutternov** [comment]
There is TWO leveractions; I removed the one that is a full stock rather than a no-stock.

I answered this in the Discord a few times already - including a certain someone asking in a dickish way instantly without literally looking up just a few lines of code where the removal is. Never change, F13.....

As for the AER-14, it was already in loot pools both in this codebase and on DR. I did not touch that. The AER-12 is not canon and its purpose is deemed moot by the Wattz 2k doing the same thing but, funnily, better as it has **less** slowdown and **more** damage yet roughly the same rarity in the loot pool (Same with the AER-9 focused which is more lore accurate.)

**Comment by Stutternov** [comment]
image
To prove my point... community is just being a complete prick without reading anything I said. Including insulting me over it. Ebic.

**Comment by Zero-Percent-Angel** [comment]
Just need to get the map CI passing and I'll merge

**Comment by BadAtThisGame302** [comment]
> There is TWO leveractions; I removed the one that is a full stock rather than a no-stock. Reason why is the one in-game in NV is the stockless variant. Similarly, they did the exact same thing. If someone ports the cosmetic gun-sprite thing then it can become an alt-sprite variant.
> 
> I answered this in the Discord a few times already - including a certain someone asking in a dickish way instantly calling me a dumbass without literally looking up just a few lines of code where the removal is. Never change, F13.....
> 
> As for the AER-14, I've only added it to the unique weapon loot-pool. AER-12 I've added back then in exchange for replacing the AER-9 Focused with it; as the AER-9 focused was - kind of objectively better even if it had a higher cell usage. Plus, the AER-12 is supposed to be unreliable due to its crystal so.. there. It's now a very good gun, acting like the Focused, but has the same drawback now of not many rounds per cell.

Sorry about the lever action thing, my Git is bugged out and I couldn't check the files then.

Great, I'd rather have the AER9 "focused" out than a slightly different and "unique" than the same gun but more powerful in the game.

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -668,11 +668,11 @@
...
-//Quirked-up AER9
-/obj/item/gun/energy/laser/aer9/focused
-	name = "\improper Hot-wired AER9 laser rifle"
-	desc = "A sturdy pre-war laser rifle. Emits beams of concentrated light to kill targets. This one has been jury-rigged against common sense to dump more power into its shots."
-	ammo_type = list(/obj/item/ammo_casing/energy/laser/lasgun/hitscan/focused)
+//AER-12 - basically stronger AP/damage AER-9 but with higher power consumption.
+/obj/item/gun/energy/laser/aer12
+	name = "AER12 laser rifle"
+	desc = "A sturdy pre-war laser rifle. Emits beams of concentrated light to kill targets. This model is an 'improvement' over the AER9 but suffers extra power consumption in return for its better focused beam."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/aer12/hitscan)
```
Comment (BadAtThisGame302) [inline code/modules/projectiles/guns/energy/laser.dm:675]: I think the AER12 has a different sprite from the AER9 you can add, not sure though!

Comment (Stutternov) [reply code/modules/projectiles/guns/energy/laser.dm:675]: Fixed

**Pull request merged**